README.md

```diff
@@ -1,24 +1,41 @@
-# README
+# Political Ties
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This is a project were are using [Rails](http://rubyonrails.org) to service the public on what's up with the political things are going on in the Traverse City region. For example, who is going to run in the next presidential election? What proposals do you have to vote on in the next election? Who's running for the TCAPS board? This service has you covered. 
 
-Things you may want to cover:
+## Requirements
 
-* Ruby version
+Some things you want to have:
 
-* System dependencies
+### Ruby version: 
+`2.2.2 or higher`
 
-* Configuration
+### System dependencies: 
+* Bundle: `gem install bundle`
```

Comment:
Would like this updated to be called it's proper name, '[Bundler](http://bundler.io)'. Though I think `gem install bundle` does in fact get the correct gem. 🤔